README.md

```diff
@@ -260,6 +260,26 @@ Proceed similar for other `.ui` files:
 1. if you edited or added new widgets you will have to change or include them in `ORStools/gui/directions_gui.py`
    as well
 
+### Translation
+Translation uses the QT Linguist for translating GUI and source code strings. All translation-related content resides in `ORStools/i18n`.
+Translation is controlled by `ORStools/gui/translate.pro`, stating all UI-forms and sourcefiles that include strings to be translated.
+To add a translation, add `orstools_<LANGUAGE_TAG>.ts` to the list of translation in this file.
+
+#### Workflow
+1. Generate the `.ts`-files (Translation Source) from `translate.pro` via
+    ```shell
+   pylupdate5 translate.pro
```

Comment:
prob should add `-noobsolete` & `-verbose` here

Reply:
what do the do?

Reply:
`-noobsolete` removes obsolete entries (that otherwise stick around in the .ts file)
`-verbose` -.- gives you verbose output, which is nice to see if something removed or added or changed

Reply:
done